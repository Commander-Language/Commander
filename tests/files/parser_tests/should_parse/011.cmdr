// Test that tuples parse properly
isTuple = (1, true, "hello");