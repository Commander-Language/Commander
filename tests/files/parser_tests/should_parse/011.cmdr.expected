(PrgmNode
	(ExprStmtNode (BinOpExprNode (BindingNode isTuple) SET (TupleExprNode (ExprsNode (IntExprNode 1) (BoolExprNode true) (StringExpressionNode (StringNode (StringExprsNode (StringExpressionNode (StringNode 'hello'))))))))))
